commons/monitor/questionnaire/questionnaire_application_usage_log.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "namespace": "org.radarcns.monitor.questionnaire",
+  "type": "record",
+  "name": "QuestionnaireApplicationUsageLog",
```

Comment:
Perhaps rename to `QuestionnaireApplicationInteractionEvent`? A single record will not make a log.